<launch>

  <param name="robot_ip" type="string" value="192.168.50.21"/>  
  <param name="robot_port" type="int" value="8000"/>  

  <node name="ardu_remoter_pub" pkg="serialdevice" type="ardu_remoter_pub" output="screen" respawn="false">
  <param name="port"       value="/dev/remote_USB"/>  
  <!-- <param name="port"       value="/dev/ttyUSB0"/>   -->
  <param name="baudrate"   value="9600"/>
  <param name="loop_rate"   value="25"/>
  </node>


</launch>



